src/shared_modules/utils/cacheLRU.hpp

```diff
@@ -118,14 +118,26 @@ class LRUCache final
      *
      * @param handler The function to be applied to each key-value pair.
      */
-    void forEach(std::function<void(const KeyType&, const ValueType&)> handler) const
+    void forEach(std::function<bool(const KeyType&, const ValueType&)> handler) const
```

Comment:
Update documentation. Iteration stops after the handler returns false

Reply:
Fixed!